examples/README.md

```diff
@@ -18,17 +18,28 @@ $ dartanalyzer analysis_options.yaml .
 
 ## How do I run example tests?
 
+Change directory into the example's folder and run Dart commands there. For
+example:
+
+```console
+$ cd examples/misc
+$ pub get
+$ pub run test  # Run VM tests
+$ pub run test -p chrome  # Run browser tests
+```
+
+## How do I run the analyzer and tests for all examples?
+
 To run both the analyzer and tests for all examples use:
 
 ```
 ./tool/analyze-and-test-examples.sh
 ```
 
-If you get a warning about test failures or analysis errors,
-you might need to update one or more analyzer results files.
-For example, update `examples/misc/analyzer-results.txt`, if
-the change applies to both stable and dev Dart releases. Otherwise,
-update only the release specific file: either
+If you get a warning about test failures or analysis errors, you might need to
```

Comment:
Just FYI, we've finally codified the convention of using semantic line breaks to avoid unnecessary diffs.

https://github.com/dart-lang/site-shared/wiki/Writing-for-Dart-and-Flutter-websites#semantic-line-breaks

Reply:
Got it. Thanks for the links!